packages/twenty-front/src/modules/databases/hooks/useGetDatabaseConnectionTables.ts

```diff
@@ -10,13 +10,13 @@ import {
 type UseGetDatabaseConnectionTablesParams = {
   connectionId: string;
   skip?: boolean;
-  refreshData?: boolean;
+  shouldFetchPendingSchemaUpdates?: boolean;
```

Comment:
why don't we remove it already? You are using it everywhere

Reply:
waiting for validation from @Bonapara that we do want the update status everywhere!

Reply:
removed it